exams/management/commands/authorization_user_exam.py

```diff
@@ -32,41 +28,29 @@ def add_arguments(self, parser):
             type=int
         )
         parser.add_argument(
-            "--course-key",
-            dest="course_key",
-            default=None,
-            help="Edx course key where we want to authorize users who passed course(s).",
-            type=int
+            "--all",
+            dest="all",
+            default='n',
+            help="Authorize eligible users for exams. Can be y or n",
+            type=str
```

Comment:
this should just be an argument without value, meaning I should be able to specify `--all` and not `--all y`